test/tests.js

```diff
@@ -0,0 +1,45 @@
+//THIS IS THE QUNIT TESTING AREA, IT IS NOT YOUR SITE JS FILE RIGHT NOW
```

Comment:
Love your personals comments on some of these files. I didn't start commenting my code until bootcamp 2 and when I did, things started to click for me more. Great jobs.

Reply:
Job*